node/common/src/service/p2p.rs

```diff
@@ -75,11 +75,7 @@ impl webrtc::P2pServiceWebrtc for NodeService {
         auth: ConnectionAuth,
     ) {
         let encrypted = auth.encrypt(&self.p2p.sec_key, other_pub_key, &mut self.rng);
-        if let Some(peer) = self.peers().get(&peer_id) {
-            let _ = peer
-                .cmd_sender
-                .send(webrtc::PeerCmd::ConnectionAuthorizationSend(encrypted));
-        }
+        Self::auth_send(self, peer_id, other_pub_key, encrypted);
```

Comment:
what is this change about?

Reply:
@binier ^

Reply:
Oh. I wonder how this got here. It's just a cosmetic change, so we can keep it or discard the commit.